docs/api/protocol.md

```diff
@@ -24,48 +24,52 @@ app.on('ready', function() {
 
-A standard scheme adheres to what RFC 3986 calls
+A standard `scheme` adheres to what RFC 3986 calls
 [generic URI syntax](https://tools.ietf.org/html/rfc3986#section-3). This
 includes `file:` and `filesystem:`.
 
-## protocol.registerFileProtocol(scheme, handler[, completion])
+### `protocol.registerFileProtocol(scheme, handler[, completion])`
 
 * `scheme` String
 * `handler` Function
-* `completion` Function
+* `completion` Function (optional)
 
-Registers a protocol of `scheme` that will send file as response, the `handler`
+Registers a protocol of `scheme` that will send the file as a response. The `handler`
 will be called with `handler(request, callback)` when a `request` is going to be
-created with `scheme`, and `completion` will be called with `completion(null)`
+created with `scheme` and `completion` will be called with `completion(null)`
 when `scheme` is successfully registered, or `completion(error)` when failed.
 
-To handle the `request`, the `callback` should be called with either file's path
-or an object that has `path` property, e.g. `callback(filePath)` or
+To handle the `request`, the `callback` should be called with either the file's
+path or an object that has a `path` property, e.g. `callback(filePath)` or
 `callback({path: filePath})`.
 
-When `callback` is called with nothing, or a number, or an object that has
-`error` property, the `request` will be failed with the `error` number you
-specified. For the available error numbers you can use, please check:
+When `callback` is called with nothing, a number, or an object that has an
+`error` property, the `request` will fail with the `error` number you
+specified. For the available error numbers you can use, please see:
 https://code.google.com/p/chromium/codesearch#chromium/src/net/base/net_error_list.h
 
-By default the scheme is treated like `http:`, which is parsed differently
-from protocols that follows "generic URI syntax" like `file:`, so you probably
-want to call `protocol.registerStandardSchemes` to make your scheme treated as
+By default the `scheme` is treated like `http:`, which is parsed differently
+from protocols that follow "generic URI syntax" like `file:`, so you probably
+want to call `protocol.registerStandardSchemes` to have your scheme treated as a
 standard scheme.
 
-## protocol.registerBufferProtocol(scheme, handler[, completion])
+### `protocol.registerBufferProtocol(scheme, handler[, completion])`
 
 * `scheme` String
 * `handler` Function
-* `completion` Function
+* `completion` Function (optional)
 
-Registers a protocol of `scheme` that will send `Buffer` as response, the
-`callback` should be called with either an `Buffer` object, or an object that
-has `data`, `mimeType`, `chart` properties.
+Registers a protocol of `scheme` that will send a `Buffer` as a response. The
+`callback` should be called with either a `Buffer` object or an object that
+has the `data`, `mimeType`, and `chart` properties.
```

Comment:
Super minor though, now that I've read further, it looks like that the `the` is consistently used when talking about object properties.